apps/discord-bot/lang/lang.en-GB.json

```diff
@@ -133,6 +133,33 @@
             "accessKeyFailed": {
                 "title": "❌ Access Denied",
                 "description": "The access key you provided is not correct. Please try again or contact an administrator."
+            },
+            "askHelp": {
+                "title": "{{COM:bot.name}} - Agent info",
+                "fields": [
+                    {
+                        "name": "Model",
+                        "value": ["MODEL-1.0"]
+                    },
+                    {
+                        "name": "Available Prompts",
+                        "value": [
+                            "Prompt 1",
+                            "Prompt 2",
+                            "Prompt 3",
+                            "Prompt 4"
+                        ]
+                    }
+                ]
+            },
+            "askResponse": {
+                "title": "**Asked:** {{QUESTION}}",
+                "fields": [
+                    {
+                        "name": "From",
+                        "value": ["<@!{{USER}}>"]
+                    }
+                ]
```

Comment:
Can we remove those info? Looks redundant to me

![image](https://github.com/user-attachments/assets/96216cf9-d8f3-41aa-92c0-48ec0ae95227)